test/cadet/assessments/answer_types/mcq_answer_test.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Cadet.Assessments.AnswerTypes.MCQAnswerTest do
+  use Cadet.ChangesetCase, async: true
+  
+  import Ecto.Changeset
+  alias Cadet.Assessments.AnswerTypes.MCQAnswer
+
+  test "valid changeset with correct choice" do
```

Comment:
Please use `valid_changesets` and `invalid_changesets` instead

Reply:
I couldn't get it to work for some weird reason.

Reply:
What's the error message? Do ask on the group instead of coming of up with hackish solution like this.

Anyway since we're gonna rework this, it doesn't matter I guess

Reply:
What is the problem with such a test though ?

Reply:
It's not DRY